node/src/Pos/Client/Txp/Util.hs

```diff
@@ -74,19 +74,27 @@ data TxRaw = TxRaw
     -- ^ Remaining money
     }
 
-data TxError
-    = TxError { unTxError :: !Text }
+data TxError =
+      NotEnoughMoney !Coin    -- ^ Parameter: how much more money is needed
+    | FailedToStabilize !Int  -- ^ Parameter: how many attempts were performed
+    | OutputIsRedeem !Address
+    | RedemptionDepleted
```

Comment:
Add comments to these two constructors too. And remove "Parameter", explanation is clear even without it.

Reply:
@volhovM I agree that the word "parameter" in this context is ugly, but I don't think it will be clearer without it. Do you have any better suggestions?

Reply:
It's perfectly clear without it.